virtcontainers/network.go

```diff
@@ -501,30 +501,50 @@ func getLinkByName(netHandle *netlink.Handle, name string, expectedLink netlink.
 	return nil, fmt.Errorf("Incorrect link type %s, expecting %s", link.Type(), expectedLink.Type())
 }
 
-// The endpoint type should dictate how the connection needs to be made
-func xconnectVMNetwork(endpoint Endpoint, connect bool, numCPUs uint32, disableVhostNet bool) error {
+// The endpoint type should dictate how the connection needs to happen.
+func xConnectVMNetwork(endpoint Endpoint, h hypervisor) error {
 	netPair := endpoint.NetworkPair()
 
+	queues := 0
+	caps := h.capabilities()
+	if caps.isMultiQueueSupported() {
+		queues = int(h.hypervisorConfig().NumVCPUs)
```

Comment:
I still don't care for this default, but its a reasonable starting point, as it matches current logic.  We should be doing this only up to a certain number (4?)

Reply:
Oh yeah I get it, but this PR should not modify the current behavior IMO, that should be part of a PR fixing this, wdyt?